src/containers/offer-details/offer-general-info/constants.tsx

```diff
@@ -0,0 +1,67 @@
+import { TFunction } from 'react-i18next'
+import CategoryIcon from '@mui/icons-material/Category'
+import BarChartIcon from '@mui/icons-material/BarChart'
+import LanguageIcon from '@mui/icons-material/Language'
+import SellIcon from '@mui/icons-material/Sell'
+import DoneIcon from '@mui/icons-material/Done'
+
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { Offer } from '~/types'
+
+const style = {
+  wrapper: {
+    display: 'flex',
+    columnGap: '5px'
+  },
+  title: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  description: {
+    typography: { xs: 'body2', sm: 'body1' }
+  }
+}
```

Comment:
Should these styles be here?
Move it out.

Reply:
fixed